### ISSUE TITLE
Feature: Change the "Show more/Show less" button to a more subtle graphic

### ISSUE THREAD

**Issue by eyand** [tuskyapp/Tusky#2920]
I'd like to be able to send more screenshots of the app of content I like. If these people don't use Mastodon and don't know what how the CW feature works, then a screenshot containing one or more CWs is really confusing to look at. 

* * * *
- Tusky Version: any
- Android Version:
- Android Device:
- Mastodon instance (if applicable):

- [ x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
I don't think we should do that, content warnings are an integral part of Mastodon and for accessibility reasons they need to be visible and clickable.
